vars/longevityPipeline.groovy

```diff
@@ -51,13 +51,21 @@ def call(Map pipelineParams) {
                name: 'prepare_stress_duration')
 
             // ScyllaDB Configuration
-	    separator(name: 'SCYLLA_DB', sectionHeader: 'ScyllaDB Configuration Selection (Choose only one from below 6 options)')
-	    string(defaultValue: '', description: 'AMI ID for ScyllaDB ', name: 'scylla_ami_id')
-	    string(defaultValue: '', description: 'GCE image for ScyllaDB ', name: 'gce_image_db')
-	    string(defaultValue: '', description: 'Azure image for ScyllaDB ', name: 'azure_image_db')
-	    string(defaultValue: '', description: 'cloud path for RPMs, s3:// or gs:// ', name: 'update_db_packages')
+            separator(name: 'SCYLLA_DB', sectionHeader: 'ScyllaDB Configuration Selection. Choose only one from AMI, GCE, Azure, ScyllaDB Repo, or BYO ScyllaDB')
+            string(defaultValue: '', description: 'AMI ID for ScyllaDB ', name: 'scylla_ami_id')
+            string(defaultValue: '', description: 'GCE image for ScyllaDB ', name: 'gce_image_db')
+            string(defaultValue: '', description: 'Azure image for ScyllaDB ', name: 'azure_image_db')
+            string(defaultValue: '', description: 'cloud path for RPMs, s3:// or gs:// ', name: 'update_db_packages')
             string(name: 'scylla_version', defaultValue: '', description: 'Version of ScyllaDB')
             string(name: 'scylla_repo', defaultValue: '', description: 'Repository for ScyllaDB')
+            string(defaultValue: '',
+                   description: (
+                       'Custom "scylladb" repo to use. Leave empty if byo is not needed. \n' +
+                       'If set then it must be proper GH repo. Example: git@github.com:personal-username/scylla.git'),
+                   name: 'byo_scylla_repo')
+            string(defaultValue: '',
```

Comment:
The 2 moved options are about GitHub info, not Scylla.
Also, they are logically tied to 3 other parameters left below:

`byo_job_path`, `byo_default_product` and `byo_default_branch`.

The proposed split looks to me more confusing than useful.

What are you trying to solve with it?